src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -489,22 +489,49 @@ For details on valid range queries, please refer to Search Matching section in C
   <data name="UpsRsFeatureDisabled" xml:space="preserve">
     <value>UPS-RS feature is disabled.</value>
   </data>
-  <data name="InvalidProcedureStepState" xml:space="preserve">
-    <value>'{0}' is not a valid procedure step state for the specified WorkitemInstanceUID '{1}'. {2}</value>
-    <comment>{0} ProcedureStepState. {1} WorkitemInstanceUID. {2} Error code</comment>
+  <data name="InvalidProcedureStepStateTransition" xml:space="preserve">
+    <value>The workitem instance with UID: '{0}' can not transition to '{1}' state: {2}.</value>
+    <comment>{0} WorkitemInstanceUID. {1} ProcedureStepState. {2} Error code</comment>
   </data>
   <data name="MismatchSopInstanceWorkitemInstanceUid" xml:space="preserve">
     <value>SOPInstanceUID '{0}' in the payload does not match the workitem query parameter '{1}'.</value>
     <comment>{0} SOPInstanceUID. {1} WorkitemInstanceUID.</comment>
   </data>
+  <data name="MismatchAffectedSopInstanceWorkitemInstanceUid" xml:space="preserve">
+    <value>AffectedSOPInstanceUID '{0}' in the payload does not match the specified WorkitemInstanceUID '{1}'.</value>
+    <comment>{0} AffectedSOPInstanceUID. {1} WorkitemInstanceUID.</comment>
+  </data>
+  <data name="MissingWorkitemInstanceUid" xml:space="preserve">
```

Comment:
same here - can use "required tag is missing"

Reply:
Done